### PR TITLE
fix: set pointer-events: none when dragging to avoid interactivity on…

### LINKED ISSUE #357: Content is selected when dragging track
**Describe the bug**
After listeners were made passive content is now selected when scrolling by dragging track.
And there is warning in the log:
```
Ignoring ‘preventDefault()’ call on event of type ‘click’ from a listener registered as ‘passive’.
```

**To Reproduce**
1. Go to https://grsmto.github.io/simplebar/
2. Scroll by dragging vertical track
3. Content is selected

**Expected behavior**
Content should not be selected when dragging track.

**Reproducible example**
https://grsmto.github.io/simplebar/

**Additional context**
There is `preventDefault` that fixes content being selected, but it is now ignored as listeners are passive.
I think most of the listeners can be made passive but not all of them.

**Your environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        | v4.2.0
| Browser          | Firefox
| npm/Yarn         | n/a
| Operating System | Arch

### PR DESCRIPTION
**Pull request by Grsmto**
… content (fix #357)

@Demivan if you want to review!

I also separated the `onPointerEvents` in 2 so we avoid the `passive: true` on unnecessary events (which was triggering an error in Chrome).

### CODE REVIEW

File: packages/simplebar/src/simplebar.js [hunk 1]
```diff
@@ -381,19 +381,17 @@ export default class SimpleBar {
...
-      'dblclick',
-      'touchstart',
-      'touchend',
-      'touchmove'
-    ].forEach(e => {
+    ['mousedown', 'click', 'dblclick'].forEach(e => {
+      this.el.addEventListener(e, this.onPointerEvent, true);
+    });
+
+    ['touchstart', 'touchend', 'touchmove'].forEach(e => {
```
Comment (Demivan) [inline packages/simplebar/src/simplebar.js]: If `preventDefault` is removed there is no need for non-pasive listeners
https://github.com/Grsmto/simplebar/blob/f95c9f8eca614797433cb3b03b35a907aeb39265/packages/simplebar/src/simplebar.js#L722-L724

Comment (Demivan) [reply packages/simplebar/src/simplebar.js]: This is not correct. React-Select from demo is not working without `preventDefault`

Comment (Grsmto) [reply packages/simplebar/src/simplebar.js]: So you think no change is needed?

Comment (Demivan) [reply packages/simplebar/src/simplebar.js]: Yeah. Think only css fix was needed. There is no warning in Chrome and Firefox now after events were separated.

**Pull request merged**